convert2rhel/unit_tests/breadcrumbs_test.py

```diff
@@ -268,3 +270,52 @@ def test_set_target_os(pretend_os):
         "name": "CentOS Linux",
         "version": "7.9",
     } == breadcrumbs.breadcrumbs.target_os
+
+
+@pytest.mark.parametrize(("telemetry_disabled", "telemetry_called"), [(True, 0), (False, 1)])
+def test_disable_telemetry(telemetry_disabled, telemetry_called, monkeypatch):
+    if telemetry_disabled:
+        monkeypatch.setenv("CONVERT2RHEL_DISABLE_TELEMETRY", "1")
+
+    _save_migration_results = mock.Mock()
+    _save_rhsm_facts = mock.Mock()
+
+    monkeypatch.setattr(breadcrumbs.breadcrumbs, "_save_migration_results", _save_migration_results)
+    monkeypatch.setattr(breadcrumbs.breadcrumbs, "_save_rhsm_facts", _save_rhsm_facts)
+    # Set to true, pretend that user was informed about collecting data
+    breadcrumbs.breadcrumbs._inform_telemetry = True
```

Comment:
Same here as the comment above (https://github.com/oamg/convert2rhel/pull/723/files#r1108824235)